localstorage in language selection

wiruwiru opened the pull request.

ivan-keeper commented on index.html:
```diff
@@ -33,9 +33,9 @@ <h1>COSCU ARMY AWARDS</h1>
-                        <img src="/placeholder.svg?height=30&width=30" alt="Kick" class="sponsor-logo">
-                        <img src="/placeholder.svg?height=30&width=30" alt="McDonald's" class="sponsor-logo">
-                        <img src="/placeholder.svg?height=30&width=30" alt="Spotify" class="sponsor-logo">
+                        <img src="https://kick.com/img/kick-logo.svg" alt="Kick" class="sponsor-logo">
+                        <img src="./img/McDonald's-Logo.wine.png" alt="McDonald's" class="sponsor-logo">
+                        <img src="https://upload.wikimedia.org/wikipedia/commons/8/84/Spotify_icon.svg" alt="Spotify" class="sponsor-logo spotify">
```
Creo que sería mejor descargar los logos y tenerlos en una carpeta en el proyecto, como hiciste con el de McDonalds

wiruwiru committed:
localstorage in language selection

The language selected by the user is now stored for future visits to the website.

wiruwiru committed:
unnecessary
